app/src/main/kotlin/com/tailoredapps/template/ui/base/navigator/ActivityNavigator.kt

```diff
@@ -63,18 +64,30 @@ open class ActivityNavigator(protected val activity: FragmentActivity) : Navigat
         startActivityInternal(activityClass, requestCode, adaptIntentFun)
     }
 
+    override fun startActivityWithTransition(activityClass: Class<out Activity>, vararg transitionViews: View?, adaptIntentFun: (Intent.() -> Unit)?) {
+        val intent = Intent(activity, activityClass)
+
+        val mapped = transitionViews
+                .filter { it != null }
```

Comment:
If you make the type `vararg View` the null filter would not be needed. Is there something I'm missing? 🤔